pyscf/adc/uadc.py

```diff
@@ -994,24 +956,21 @@ def outcore_transform():
 
         eris = self.transform_integrals() 
 
-        if self.compute_mpn_energy == True:
-            self.e_corr, self.t1, self.t2 = compute_amplitudes_energy(self, eris=eris, verbose=self.verbose)
-            self._finalize()
-        else:
-            self.t1, self.t2 = compute_amplitudes(self, eris=eris)
-            self.e_corr = None
+        self.e_corr, self.t1, self.t2 = compute_amplitudes_energy(self, eris=eris, verbose=self.verbose)
+        self._finalize()
 
         self.method_type = self.method_type.lower()
         if(self.method_type == "ea"):
-            e_exc, v_exc, spec_fac = self.ea_adc(nroots=nroots, guess=guess, eris=eris)
+            e_exc, v_exc, spec_fac, X_a, X_b, adc_es = self.ea_adc(nroots=nroots, guess=guess, eris=eris)
```

Comment:
As mentioned before, X should be passed as a tuple of two arrays. Please make changes everywhere.

Reply:
Done